src/Pico_Led.py

```diff
@@ -22,6 +22,7 @@ def start_fast_blink():
 def start_slow_blink():
     timer.init(freq=1, mode=machine.Timer.PERIODIC, callback=toggle_led)
 
+#TODO not called anywhere but also 1/3 is integer division
 def start_long_short_blink():
     timer.init(freq=1/3, mode=machine.Timer.PERIODIC, callback=toggle_led)
```

Comment:
This doesn't get used but it's integer division which could be an issue.

Reply:
should remove that function - the other normal blink functions are used and do work